library/Recurly/V3/API/Account.hs

```diff
@@ -26,7 +26,7 @@ getAccountBillingInfo
 getAccountBillingInfo accountCode = Recurly.liftRecurly $ do
   request <- RecurlyApi.makeRequest
     ["accounts", into @PathPiece.PathPiece accountCode, "billing_info"]
-  RecurlyApi.sendRequestList request
+  RecurlyApi.sendRequest request
```

Comment:
@saramuse does this change make sense? Or should we update the function to return a list of billing infos?

Reply:
Yes it does! Should just be a single billing info, good catch!